### PR TITLE
Set and get structural plasticity status parameters through set and getKernelStatus

### PR DESCRIPTION
**Pull request by sdiazpier**
This PR addresses Issue #777 and removes the additional functions to set and get structural plasticity parameters. This is now done through set and getKernelStatus.

### CODE REVIEW

**Comment by heplesser** [comment]
@lekshmideepu It seems that CI has not run for this PR. Is there any way to make Travis run it?

**Comment by lekshmideepu** [comment]
> @lekshmideepu It seems that CI has not run for this PR. Is there any way to make Travis run it?

Restarted the build just now.

**Comment by lekshmideepu** [comment]
> > @lekshmideepu It seems that CI has not run for this PR. Is there any way to make Travis run it?
> 
> Restarted the build just now.

@heplesser  Travis build failed with this [error.](https://travis-ci.org/github/nest/nest-simulator/jobs/678263282#L5713)

**Comment by heplesser** [comment]
> > > @lekshmideepu It seems that CI has not run for this PR. Is there any way to make Travis run it?
> > 
> > 
> > Restarted the build just now.
> 
> @heplesser Travis build failed with this [error.](https://travis-ci.org/github/nest/nest-simulator/jobs/678263282#L5713)

@sdiazpier The Travis build seems to have failed because there is still at least one `GetStructuralPlasticityStatus()` call left somewhere, probably in a test. Please also double-check all examples and documentation!

@lekshmideepu The test failed not gracefully due to the way `do_tests.sh` currently parses its output, see #1548.

**Comment by heplesser** [comment]
@sdiazpier What is the status on this one?

**Comment by terhorstd** [comment]
@sdiazpier friendly ping from the hackathon

**Comment by terhorstd** [comment]
This seems to break because of `junitparser` not being installed by travis. see #1578 and #1652

File: pynest/nest/tests/test_sp/test_get_sp_status.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ class TestGetStructuralPlasticityStatus(unittest.TestCase):
     assert ('structural_plasticity_update_interval' in all)
     assert (all['structural_plasticity_update_interval'] == 10000.)
 
-    sp_synapses = nest.GetStructuralPlasticityStatus(
+    sp_synapses = nest.GetKernelStatus(
         'structural_plasticity_synapses'
     )
```
Comment (terhorstd) [inline pynest/nest/tests/test_sp/test_get_sp_status.py:78]:
```suggestion
    sp_synapses = nest.GetKernelStatus( 'structural_plasticity_synapses' )
```

**Comment by terhorstd** [comment]
PyNEST tests seem to fail
```
Phase 7: Running PyNEST tests.
------------------------------

  Failure: AttributeError (module 'nest' has no attribute 'GetStructuralPlasticityStatus') ... ERROR
  ERROR: Failure: AttributeError (module 'nest' has no attribute 'GetStructuralPlasticityStatus')
    File "/home/travis/virtualenv/python3.6.10/lib/python3.6/site-packages/nose/failure.py", line 39, in runTest
  AttributeError: module 'nest' has no attribute 'GetStructuralPlasticityStatus'
```

**Comment by stinebuu** [comment]
@sdiazpier I think the failing test comes from [test_get_sp_status.py](https://github.com/sdiazpier/nest-simulator/blob/sp-kernel/pynest/nest/tests/test_sp/test_get_sp_status.py), which tests `GetStructuralPlasticityStatus`. As this PR removes the founction, I think you can you delete the test!

**Review by heplesser — APPROVED** [review]

**Pull request merged**